### PR TITLE
Include lpms min enc resolution

### PR DESCRIPTION
**Pull request by AlexKordic**
**What does this pull request do? Explain your changes. (required)**

Include changes from [PR 326](https://github.com/livepeer/lpms/pull/326).

**Specific updates (required)**

- Update LPMS dependency version.
- Fix usage of `GetCodecInfoBytes()` to match return types.
- Include all LPMS commits including d82aa89e247702de2e1eed9a352c18fb4c459367

**How did you test each of these updates (required)**

**Does this pull request close any open issues?**

**Checklist:**

- [x] Read the [contribution guide](./doc/contributing.md)
- [x] `make` runs successfully
- [x] All tests in `./test.sh` pass
- [x] README and other documentation updated
- [x] [Pending changelog](./CHANGELOG_PENDING.md) updated

### CODE REVIEW

File: CHANGELOG_PENDING.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
 #### Transcoder
 
 ### Bug Fixes 🐞
+- [task-runner \#24](https://github.com/livepeer/task-runner/issues/24) rendition resolution fix for portrait input videos; Min resolution applied for Nvidia hardware (@AlexKordic)
```
Comment (thomshutt) [inline CHANGELOG_PENDING.md]: Should this link to the PR rather than / as well as the issue it fixes?

**Review by thomshutt — APPROVED** [review]

File: CHANGELOG_PENDING.md (hunk 1 above)
Comment (AlexKordic) [reply CHANGELOG_PENDING.md]: I am not sure, being another project. What do you think how to verbalize this item?

Comment (thomshutt) [reply CHANGELOG_PENDING.md]: Yeah, that is a little unusual - I think include the PR ID at the start like normal and then have the task-runner URL somewhere in the message

Comment (AlexKordic) [reply CHANGELOG_PENDING.md]: Fixed to look normal

**Review by thomshutt — APPROVED** [review]

**Pull request merged**